libs/platform/src/lib/components/form/date-picker/date-picker.component.html

```diff
@@ -0,0 +1,38 @@
+<div fd-form-item>
```

Comment:
it should not be wrapped as we use formField for it

Reply:
removed.
i wanted to ask 1 Question.
In Checkbox you had asked to wrap <fd-checkbox> with "fd-form-item". what is the use of this wrapping?
is this because checkbox getting used outside form without use of "ngModel" ?